mlc_config.json

```diff
@@ -0,0 +1,40 @@
+{
+    "ignorePatterns": [
+        {
+            "pattern": "http://127.0.0.1"
+        },
+        {
+            "pattern": "https://www.codegrepper.com"
+        },
+        {
+            "pattern": "https://crates.io"
+        },
+        {
+            "pattern": "http://deum-longis.org"
+        },
+        {
+            "pattern": "http://ergohoc.com"
+        },
+        {
+            "pattern": "http://www.in-ulla.io"
+        },
+        {
+            "pattern": "http://inplevere-virum.net"
+        },
+        {
+            "pattern": "http://latonam.com"
+        },
+        {
+            "pattern": "http://localhost"
+        },
+        {
+            "pattern": "http://moleorbataque.io"
+        },
+        {
+            "pattern": "http://occideratparem.io"
+        },
+        {
+            "pattern": "http://pedesque-huius.org"
```

Comment:
where is this list from? `localhost` makes sense, but the rest?

Reply:
Looks like these are generated by the lorem ipsum generator and ended up here https://github.com/roc-lang/roc/blob/main/examples/static-site-gen/input/apple.md . We should probably replace all those with a link to the roc website or something. Allowing a bunch of random links in the repo does not set a good precedent.

Reply:
I'll take care of that.